src/Redux/Store/selectors.ts

```diff
@@ -11,7 +12,7 @@ export const getUpgradedBasicNemesisCardsByExpeditionId = createSelector(
     Settings.selectors.Expansions.Languages.getLanguagesByExpansion,
   ],
   (expedition, content, languages) =>
-    expedition.upgradedBasicNemesisCards.map(id =>
+    expedition.upgradedBasicNemesisCards.map((id) =>
```

Comment:
All these edits seem to come from husky prettifying the file because I touched it.

Reply:
Ah thanks for mentioning this - this is probably fine. I'll have a look asap